zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/RegisteredInterpreter.java

```diff
@@ -0,0 +1,100 @@
+package org.apache.zeppelin.interpreter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represent registered interpreter class
+ */
+public class RegisteredInterpreter {
```

Comment:
Seems safe to assume that this is not original code but moved from existing implementation?